src/components/AcceptOrRejectAllOrMoreChoices.tsx

```diff
@@ -50,16 +50,21 @@ export function AcceptOrRejectAllOrMoreChoices({
   };
 
   return (
-    <div className="column-content">
+    <div className="column-content" role="none">
```

Comment:
we have a dialog role on a parent element here! also focus trapping is specifically [against WCAG A guidelines](https://www.w3.org/TR/WCAG22/#no-keyboard-trap), which is what im working off of here

Reply:
Fascinating. It says the opposite [here](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/#:~:text=If%20focus%20is%20on%20the%20last%20tabbable%20element%20inside%20the%20dialog%2C%20moves%20focus%20to%20the%20first%20tabbable%20element%20inside%20the%20dialog.).

Reply:
lol, hm. if wcag itself is split on it then id probably err on the side of doing less work here and not trapping focus...? what do you think

Reply:
Ah, I think that guideline is actually saying the user shouldn't ever get permanently stuck in one location with no way to return to the main page. 

[Here](https://www.w3.org/WAI/WCAG21/Understanding/no-keyboard-trap.html#:~:text=the%20applet%20itself.-,A%20modal%20dialog%20box,-A%20Web%20application) are some further guidelines and they address modals.

Reply:
ah, alright. they still dont say that modal need to be focus trapped to meet WCAG A criteria though, and since thats the scope of this refactor ill probably leave off the focus trapping. happy to make a ticket for it if youd like though!

Reply:
Up to you! Doesn't seem necessary for me, unless you anticipate needing to meet higher requirements later, in which case you might as well capture the recommendation. Does seem like this might be a "pattern to follow" more than a requirement? But I haven't looked at the specific requirements in ages.

Reply:
i didnt see anything about it in their AA or AAA criteria either, so maybe just more of a pattern? ill leave a note about it in a comment in case we ever want to meet a11y guidelines